runtime/src/lib.rs

```diff
@@ -767,6 +767,53 @@ impl substrate_pallet_multi_account::Trait for Runtime {
     type MaxSignatories = MultiAccountMaxSignatories;
 }
 
+parameter_types! {
+	pub const SpendPeriod: BlockNumber = 6 * DAYS;
```

Comment:
What does the SpendPeriod do? Is it how often they can be issued?

Reply:
Yes, it is the period between successive spends.